### PR TITLE
md: Wait for block device to go away after stopping

### PR DESCRIPTION
**Pull request by lsgunth**
Turns out this patch is actually going to be required.

The fix that's being worked on doesn't fix this issue, it's considered a userspace issue to avoid reusing used devices.

### CODE REVIEW

**Comment by dlsloan** [comment]
Testing this on my setup with dsloan/nvme changes applied, looks good assuming nothing pops there

File: md.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 import pathlib
 import re
 import subprocess
+import sys
```
Comment (dlsloan) [inline md.py]: Why sys? I don't see it used here.

Comment (lsgunth) [reply md.py]: Fixed

**Review by dlsloan — APPROVED** [review]
Seems to fix/work arround my issues

**Pull request merged**